### PR TITLE
kubevirt: Add aliases 'interface_name' for 'network_name'

### PR DESCRIPTION
**Pull request by Akasurde**
##### SUMMARY

Fixes: ansible/ansible#55641

Signed-off-by: Abhijeet Kasurde <akasurde@redhat.com>

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
changelogs/fragments/55903_kubevirt.yml
plugins/inventory/kubevirt.py

### CODE REVIEW

**Comment by Akasurde** [comment]
@machacekondra @mmazur @pkliczewski Do you think we should make this aliases or deprecate `network_name` in favor of `interface_name`? Let me know.

File: changelogs/fragments/55903_kubevirt.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+---
+bugfixes:
```
Comment (felixfontein) [inline changelogs/fragments/55903_kubevirt.yml:2]: Shouldn't that be:
```suggestion
minor_changes:
```
? I don't see how this is a bugfix.

Comment (Akasurde) [reply changelogs/fragments/55903_kubevirt.yml:2]: This parameter was supposed to be `interface_name`. I am waiting for Kubevirt maintainers to comment and will update accordingly.

**Comment by mmazur** [comment]
@Akasurde unless there's a good reason to break compatibility, alias is better.

**Comment by mmazur** [comment]
LGTM

**Comment by felixfontein** [comment]
@Akasurde thanks for fixing this!
@mmazur thanks for reviewing!

**Pull request merged**